spring-boot/src/main/resources/db/migration/V1__Create_people_table.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE people (
+    nickname VARCHAR(32) NOT NULL UNIQUE,
+    name VARCHAR(100) NOT NULL,
+    birthdate DATE NOT NULL,
+    skills_stack TEXT []
+);
```

Comment:
nitpick: podemos chamar só de `stack`.
```suggestion
    stack TEXT []
);

```